tools/gn

```diff
@@ -1070,6 +1102,19 @@ def parse_args(args):
       help='Do not run GN. Instead configure the Impeller cmake example build.',
   )
 
+  parser.add_argument(
```

Comment:
Are we giving too much power to this script? by the name it seems it was originally designed as a gn wrapper but now it is doing gn, cmake, and goma.

Reply:
By design, this is the build setup step that the engine_v2 recipe calls, so if we want to run something before we do a ninja build, this is where it has to go, right?